License format on headers

DIOHz0r opened the pull request:
### Changes description

Small fix of the license tag for headers

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [ ] Docs have been added/updated

### Estimated time

<!-- Add the number of pomodoros spent on this task -->

|Assignee|:tomato:|
|:---|:---:|
|@DIOHz0r |1|

<!--- Task not finished? Please give an update of the time --->

### References

<!-- issues related (for reference or to be closed), dependencies and/or links of discuss -->

Related #N/A

DIOHz0r committed:
docs: update wrong license format on headers

Signed-off-by: Domingo Oropeza <doropeza@teclib.com>

btry reviewed (approved):
same PR must be done on develop branch due to new files introduced for FCM, despite this change was already done

btry committed:
docs(headers): remove author from headers

author is saved in each commit in the CVS, this is more granular
and easier to maitain

Signed-off-by: Thierry Bugier <tbugier@teclib.com>

btry commented:
to squash into a single commit before merge !
